src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -468,6 +481,43 @@ class BinaryCompactObject : public DomainCreator<3> {
     using group = RotationAboutZAxisMap;
   };
 
+  struct SizeMap {
+    static constexpr Options::String help = {
+        "Options for a time-dependent size maps."};
+  };
+
+  /// \brief Initial values for functions of time for size maps for objects A,B.
+  struct InitialSizeMapValues {
+    using type = std::array<double, 2>;
+    static constexpr Options::String help = {
+        "SizeMapA, SizeMapB values at initial time."};
+    using group = SizeMap;
+  };
```

Comment:
How are these options handled depending on whether or not the centers are excised?

Reply:
If a center is not excised, then the corresponding option is ignored, since the size map is not applied

Reply:
Document that somewhere.

Reply:
Documented as \details in each size map tag's documentation